task_processing/plugins/mesos/mesos_executor.py

```diff
@@ -68,6 +68,10 @@ def __invariant__(conf):
                     factory=float,
                     mandatory=False,
                     invariant=lambda t: (t > 0, 'timeout > 0'))
+    retries = field(type=int,
+                    factory=int,
+                    mandatory=False,
+                    invariant=lambda r: (r > 0, 'retries > 0'))
```

Comment:
**re**tries reads as "number of tries after failure", constraining this to `> 0` means users have no way to say "don't retry this even once". change constrain to `>=`?